pageserve/app.py

```diff
@@ -76,29 +76,19 @@ def parsed_posts(posts):
 
 def get_events():
     """Gets all sub-events from events service."""
-    # TODO(cmei4444): integrate with events service to pull event info from
-    # database
-    events = [{'event_id': '1',
-               'name': 'concert 1',
-               'description': 'listen to fun music here!',
-               'author': 'admin',
-               'created_at': '7-9-2019',
-               'event_time': '7-10-2019',
-               },
-              {'event_id': '2',
-               'name': 'concert 2',
-               'description': 'listen to fun music here!',
-               'author': 'admin',
-               'created_at': '7-9-2019',
-               'event_time': '7-12-2019',
-               }]
-    return parsed_events(events)
-
-
-def parsed_events(events):
+    url = os.environ.get("EVENTS_ENDPOINT")
+    r = requests.get(url, params={})
+    if r.status_code == 200:
+        return parsed_events(r.json())
+    else:
+        # TODO(cmei4444): handle error in a way that doesn't break page display
+        return "Error in getting events"
+
+
+def parsed_events(events_dict):
```

Comment:
Add docstring to explain what this does and maybe why you need to extract the `'events'` list from `events_dict`.

Reply:
Added in 5dfe2be